### DISCUSSION TITLE
Why Button does not receive Pointer* events ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [microsoft/microsoft-ui-xaml#7849] [Q&A]
```
<Button x:Name="LegendButton" Background="DarkMagenta" 
            PointerPressed="OnPointerPressed" 
            PointerReleased="OnPointerReleased" 
            PointerMoved="OnPointerMove" 
            PointerCanceled="OnPointerReleased">
        <Button.TranslationTransition>
            <Vector3Transition>
            </Vector3Transition>
        </Button.TranslationTransition>
</Button>
```

**Comment by marcelwgn** [comment]
In my testing pointer events like `PointerMoved` or `PointerExited` work as expected and get raised. Events that handle "activation" are generally not available since you should use the `Click` event as pointed out by the documentation: https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.button?view=windows-app-sdk-1.1&WT.mc_id=WDIT-MVP-5004239#button-interaction
